src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -77,19 +78,15 @@ public void run(String[] command) throws ProcessRunnerException {
       }
       processBuilder.command(makeOsSpecific(command));
 
-      synchronized (this) {
```

Comment:
I like this solution. It felt a bit restrictive before to only allow a single process at a time per sdk.